### PR TITLE
Prepare for the next release

### PR DESCRIPTION
**Pull request by rtpg**
I'm not sure who is the release manager for this lib, and I don't see any "autorelease on tag" infrastructure, so I'm going to ping both @auvipy and @jdufresne on this (apologies in advance to the one who didn't need to get pinged here)

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,13 +1,17 @@
 Changelog
 =========
 
-UNRELEASED
-~~~~~~~~~~
+1.2.1 (2020-05-19)
```
Comment (jdufresne) [inline CHANGELOG.rst:4]: I think we should go to 1.3.0. WDYT?

Comment (rtpg) [reply CHANGELOG.rst:4]: Hesitated on this point, let's go with  1.3.0! I'll send in the changes

Comment (rtpg) [reply CHANGELOG.rst]: Alright 1.3.0 it has become

**Comment by jdufresne** [comment]
Thanks. Please feel free to merge and release at your convenience.

**Comment by rtpg** [comment]
Alright I pushed up the tag, my understanding is that this should trigger something over on your end as release managers. Thanks for the super fast response times :100:

**Pull request merged**